test/Microsoft.TemplateEngine.Core.UnitTests/StubSettingsLoader.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.TemplateEngine.Abstractions;
+using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Abstractions.Mount;
+
+namespace Microsoft.TemplateEngine.Core.UnitTests
+{
+    internal class StubSettingsLoader : ISettingsLoader
+    {
+        public IComponentManager Components => throw new NotImplementedException();
+
+        public IEngineEnvironmentSettings EnvironmentSettings => throw new NotImplementedException();
+
+        public IEnumerable<MountPointInfo> MountPoints => throw new NotImplementedException();
+
+        public void AddMountPoint(IMountPoint mountPoint)
+        {
+            throw new NotImplementedException();
```

Comment:
I had thought that if no one has needed it thus far they probably wouldn't going forward and YAGNI.  Is it acceptable to have empty methods or return default values for all methods?  The other reason I didn't fully implement is I'm less familiar with this class and how to fully mock it out for testing purposes.